velox/exec/RowContainer.h

```diff
@@ -823,7 +823,8 @@ class RowContainer {
 
   /// Returns true if specified column may have nulls, false otherwise.
   inline bool columnHasNulls(int32_t columnIndex) const {
-    return columnHasNulls_[columnIndex];
+    return columnStats(columnIndex).has_value() &&
```

Comment:
```
s/columnHasNulls/columnMayHaveNulls/
!columnStats(columnIndex).has_value() || columnStats(columnIndex)->nullCount() > 0?
```

Reply:
As @tanjialiang suggested, I'll separate the nullCount when invalidating columnStats, so that we don't need to check it here.